tcf_website/management/commands/index_elasticsearch.py

```diff
@@ -1,6 +1,4 @@
-"""
-Modules used
```

Comment:
Maybe "Management command that loads courses and instructors into Elasticsearch"

Reply:
done